pipeline/metadata/flatten.py

```diff
@@ -15,6 +15,9 @@
 from pipeline.metadata.flatten_satellite import SatelliteFlattener, SATELLITE_PATH_COMPONENT
 from pipeline.metadata.flatten_hyperquack import HyperquackFlattener
 
+# Randomly chosen UUID used as a namespace for generating further UUIDs
+CENSORED_PLANET_NAMESPACE = uuid.UUID('23fd299b-6547-4dca-b630-cb05dec70e0e')
```

Comment:
You could also use something like `uuid.uuid5(uuid.NAMESPACE_DNS, "censoredplanet.org")`